### DISCUSSION TITLE
find/till use cases

### DISCUSSION THREAD

**Discussion by drdrjacobs** [meow-edit/meow#378] [Q&A]
I've been playing with Meow for a little while now, and I pretty much never use find/till functionality. However, these commands are given prominent placements in the default keybinding files, suggesting that they must be pretty commonly used in theory.

What are the use cases that I am missing? Or am I not alone in rarely reaching for these commands?

**Comment by gtusr** [comment]
Typing the target letter of your operation is often easier than referencing it indirectly, as in "the fifth word to the right of the cursor," for example. 

That said, it can become even more useful when coupled with incremental search.  For example, if there are several instances of the letter 'y' between the cursor and where I would like to jump, it is nice to have two options available after the cursor jumps to the first 'y' it finds:  (1) hit <return> to exit if this instance of 'y' is the desired one or (2) type the next letter that appears after the desired 'y' to (hopefully) move to the correct one.  

Another possibility is just to use a key like <tab> to step forward to additional instances of the letter 'y' 

@DogLooksGood:   I'm not all that familiar with Meow yet, so if it doesn't already exist in the package, are there any plans to add this "incremental update" or "tab forward" feature to the find/till commands?

**Comment by DogLooksGood** [comment]
The "default key bindings" is just an example, you can adjust according to your preference. 

For the use case of find/till,
- I use `find` to select to a close paren or a comma or a beginning of a word(because there's no Vim's `w` in Meow). 
- I use `till` to select till the semicolon in languages like C/JS/Rust, or the end of the line. 
- I also use them in BEACON for formatting texts.
